### PR TITLE
Variance statement 

### PR DESCRIPTION
**Pull request by dbojicic**
Updates to variance statement to 
- add AU Sex Assigned at Birth profile as agreed in [2024-07-03 FHIRWG Meeting Minutes](https://confluence.hl7.org/display/HAFWG/2024-07-03+FHIRWG+Meeting+Minutes)
- remove Observations profiles as the Observation resource is profiled in AU Base (only where a resource is not profiled in AU Base should those additional profiles be added)

### CODE REVIEW

File: input/pagecontent/variance.md [hunk 1]
```diff
@@ -4,13 +4,5 @@ This implementation guide has no variance (i.e. fully compliant) from AU Base FH
...
-  - [AU Core Body Height](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-bodyheight.html)
-  - [AU Core Body Temperature](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-bodytemp.html)
-  - [AU Core Body Weight](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-bodyweight.html)
-  - [AU Core Diagnostic Result Observation](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-diagnosticresult.html)
-  - [AU Core Heart Rate](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-heartrate.html)
-  - [AU Core Respiration Rate](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-resprate.html)
-  - [AU Core Smoking Status](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-smokingstatus.html)
-  - [AU Core Waist Circumference](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-waistcircum.html)
+- Extension
+  - [AU Core Sex Assigned At Birth](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-rsg-sexassignedab.html)
```
Comment (dt-r) [inline input/pagecontent/variance.md:8]: add to the end of it "profiles core FHIR extension ..." and replace ... with the RSG extension.

File: input/pagecontent/variance.md [hunk 2]
```diff
@@ -4,13 +4,5 @@ This implementation guide has no variance (i.e. fully compliant) from AU Base FH
 #### Additionally Profiled Resources
```
Comment (dt-r) [inline input/pagecontent/variance.md:4]: At this time is makes sense to add in the introductory text to this page to explain what this page does. We would then submit a PR to eReq to incorporate the same text.

**Review by dt-r — CHANGES_REQUESTED** [review]
Enhancements requested. See inline comments.

File: input/pagecontent/variance.md (hunk 2 above)
Comment (dbojicic) [reply input/pagecontent/variance.md:15]: Added description to the page

File: input/pagecontent/variance.md (hunk 1 above)
Comment (dbojicic) [reply input/pagecontent/variance.md]: Done

**Review by dt-r — APPROVED** [review]

**Pull request merged**